rfcs/0000-rlp-status.md

```diff
@@ -0,0 +1,162 @@
+# RFC Template
+
+- Feature Name: `policy_status_states`
+- Start Date: 2023-02-03
+- RFC PR: [Kuadrant/architecture#0009](https://github.com/Kuadrant/architecture/pull/9)
+- Issue tracking: [Kuadrant/architecture#0000](https://github.com/Kuadrant/architecture/issues/0000)
+
+# Summary
+[summary]: #summary
+
+This RFC proposes a new design for any Kuadrant Policy (`RateLimitPolicy`, `AuthPolicy`, etc..) Status definition and transitions.
+
+# Motivation
+[motivation]: #motivation
+
+At the time being, the `RateLimitPolicy` and `AuthPolicy` Status doesn't clearly and truthfully communicate the actual state of
+reconciliation and healthiness with its Operator managed services, mainly in this case the Rate Limit service `Limitador`, and
+the Auth service `Authorino`.
+
+As a consequence, only misleading information is shared causing unexpected errors and flawed assumptions.
+
+The following are some issues addressing the before mentioned drawbacks:
+
+* https://github.com/Kuadrant/kuadrant-operator/issues/87
+* https://github.com/Kuadrant/kuadrant-operator/issues/96
+* https://github.com/Kuadrant/kuadrant-operator/issues/140
+
+# Guide-level explanation
+[guide-level-explanation]: #guide-level-explanation
+
+The following proposal it's divided in 3 stages, where each one could be applied/develop in order and would reflect valuable
+and accurate information with different degrees of acuity. The Policy CRD Status in the following diagrams, are simplified
+as states, which in the [Reference-level explanation](#reference-level-explanation) will be translated to the actual Status Conditions.
+
+## Stage 1
+**The state is defined by the application and validation of the Policy CR**
```

Comment:
Yes, it basically is about the number 4 you mention above. I didn't mind anything below the admission control stage, but if we're gonna have our own set of rules for admission control, it might reflect this in the state